src/hwas/hwas.c

```diff
@@ -178,6 +178,41 @@ MCAN1_Handler(void)
     }
 }
 
+void
```

Comment:
How does it affect using TASTE Uart driver?

Reply:
Fixed in new commit